src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -52,12 +72,14 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
       pathPrefix("workspaces") {
         pathEnd {
           requireUserInfo() { _ =>
-            passthrough(rawlsWorkspacesRoot, HttpMethods.GET)
+            mapHttpResponse(transformListWorkspaceRequest) {
+              passthrough(rawlsWorkspacesRoot, HttpMethods.GET)
+            }
           } ~
           post {
             requireUserInfo() { _ =>
               entity(as[WorkspaceCreate]) { createRequest => requestContext =>
-                val extReq = Post(FireCloudConfig.Rawls.workspacesUrl, createRequest)
+                val extReq = Post(FireCloudConfig.Rawls.workspacesUrl, WorkspaceCreate.toWorkspaceRequest(createRequest))
```

Comment:
This had to come back explicitly (also for clone) because there's no reason for the compiler to go looking for an implicit type conversion.